codec/src/main/java/io/netty/handler/codec/BinaryHeaders.java

```diff
@@ -17,67 +17,67 @@
 package io.netty.handler.codec;
 
 /**
- * A typical {@code AsciiString} multimap used by protocols that use binary headers (such as HTTP/2) for the
- * representation of arbitrary key-value data. {@link AsciiString} is just a wrapper around a byte array but provides
+ * A typical {@code ByteString} multimap used by protocols that use binary headers (such as HTTP/2) for the
+ * representation of arbitrary key-value data. {@link ByteString} is just a wrapper around a byte array but provides
  * some additional utility when handling text data.
  */
-public interface BinaryHeaders extends Headers<AsciiString> {
+public interface BinaryHeaders extends Headers<ByteString> {
     /**
      * A visitor that helps reduce GC pressure while iterating over a collection of {@link Headers}.
      */
-    interface EntryVisitor extends Headers.EntryVisitor<AsciiString> {
+    interface EntryVisitor extends Headers.EntryVisitor<ByteString> {
```

Comment:
Would it make sense to define `EntryVisitor` and `NameVisitor` to be in the `Headers` interface instead?